### ISSUE TITLE
Documentation does not work

### ISSUE THREAD

**Issue by halotukozak** [edvin/tornadofx#1362]
The [docs.tornadofx.io](https://docs.tornadofx.io) webpage does not work, so it's impossible to read the documentation.

Temporary solution: use [web.archive.org](https://web.archive.org/web/20230201162633/https://docs.tornadofx.io/)

**Comment by AJRego** [comment]
You can also check it out at [gitbooks.io](https://edvin.gitbooks.io/tornadofx-guide/content/part1/0_Subsection.html)

**Comment by chikega** [comment]
This begs the question: has development ceased for TornadoFX?

**Comment by edvin** [comment]
Yes, I'm not maintaining TornadoFX anymore, see top of README :)

**Comment by chikega** [comment]
Funny that KotlinFX has the following: 
_**"NOTE: This project is unmaintained. Please check out TornadoFX as an up-to-date and much more comprehensive alternative."** _
I guess there's really just JavaFX that's being currently maintained for desktop applications.
